app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -35,7 +35,9 @@ image_2_alt: "",
 <li class="tag">Uses of technology</li>
 <li class="tag">Online safety </li>
 </ul>
-<p>You'll teach 14 to 16 year olds (key stage 4) computer science as a GCSE qualification, teaching pupils ways to protect their online privacy and identity. Themes you might also cover include:</p>
+<p>When you're teaching key stage 3 you could use interactive exercises during your lessons, like creating a robot car.</p>
+<p><a href="https://www.thenational.academy/teachers/programmes/computing-secondary-ks3-l/units">Explore what teaching a key stage 3 computing lesson would be like</a>.</p>
+<p>Themes you'll cover for 14 to 16 year olds (key stage 4), computer science as a GCSE qualification, include:</p>
```

Comment:
```suggestion
<p>Themes you'll cover teaching 14 to 16 year olds (key stage 4) computer science as a GCSE qualification include:</p>
```
I think this sentence needs tweaking a bit (not necessarily how I've done suggested!) as the middle part feels a bit awkward/hard to understand?